Index: pkg-gpaint/gpaint.glade
===================================================================
--- pkg-gpaint.orig/gpaint.glade	2009-12-22 20:56:13.000000000 -0200
+++ pkg-gpaint/gpaint.glade	2009-12-22 20:56:33.000000000 -0200
@@ -1004,7 +1004,6 @@
 	      <property name="height_request">55</property>
 	      <property name="visible">True</property>
 	      <property name="orientation">GTK_ORIENTATION_HORIZONTAL</property>
-	      <property name="toolbar_style">GTK_TOOLBAR_BOTH</property>
 	      <property name="tooltips">True</property>
 	      <property name="show_arrow">True</property>
 
